StringUtils.c

```diff
@@ -26,6 +27,27 @@ in the source distribution for its full text.
  * at compile time (e.g. when they are immutable string literals). :)
  */
 
+#ifndef HAVE_STRCASESTR
+// OpenBSD strcasestr
```

Comment:
If this came from OpenBSD, you need to include the BSD license it was under somewhere in the code.